src/main/java/friday/logic/parser/DeleteCommandParser.java

```diff
@@ -21,7 +22,7 @@ public DeleteCommand parse(String args) throws ParseException {
             return new DeleteCommand(index);
         } catch (ParseException pe) {
             throw new ParseException(
-                    String.format(Messages.MESSAGE_INVALID_COMMAND_FORMAT, DeleteCommand.MESSAGE_USAGE), pe);
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteCommand.MESSAGE_USAGE), pe);
```

Comment:
I like the change, but it might be more work to make this consistent in other classes.